### ISSUE TITLE
Wrong version numbers in release 2.3.26.ayg

### ISSUE THREAD

**Issue by jphhoeks** [albfernandez/jboss-seam#160]
I just found your project, you are doing a great job!

I'm trying release 2.3.26.ayg, but not all version numbers are changed in this release. 

/pom.xml:
<version.seam>2.3.26-SNAPSHOT</version.seam>

/bom/pom.xml:
<version.seam>2.3.26-SNAPSHOT</version.seam>

This makes it impossible to use the BOM. Do you have a release 3.2.27.ayg planned?

**Comment by stolp** [comment]
My assumption is that there was a whoopsie with this particular release to maven central.

The intended release number would be 2.3.26.ayg instead of 3.2.26.ayg. This release is also on maven central and does work for me.

Since maven central does not allow to delete released artifacts you will need to keep this in mind when searching for new releases.

**Comment by jphhoeks** [comment]
I noticed the wrong version number 3.2.26.ayg. Unfortunately, I also made the same mistake typing this message. I have edited it.

But I do have problems using 2.3.26.ayg. I get errors that version 2.3.26-SNAPSHOT can't be found. That is correct, because those are not available on the maven repo.

If I look into 2.3.25.ayg the version in the mentionded poms is also 2.3.25.ayg, not 2.3.25-SNAPSHOT. Therefor, I guess those versions are not changed when preparing for release 2.3.26.ayg.
